index.html

```diff
@@ -20,28 +20,38 @@
         <button class="hamburger">&#9776;</button>
         <div class="nav-list">
           <ul>
-            <li><a href="#">About</a></li>
-            <li><a href="#">Projects</a></li>
-            <li><a href="#">Skills</a></li>
+            <li class="tab" data-content="aboutMe"><a href="#">About</a></li>
+            <li class="tab" data-content="projectsContainer"><a href="#">Projects</a></li>
             <li><a href="https://github.com/sheridp2" target="_blank">GitHub</a></li>
             <li><a href="https://www.facebook.com/patrick.sheridan.773?ref=bookmarks" target="_blank">Facebook</a></li>
           <ul>
         </div>
       </nav>
 
     </header>
-
-    <div class="about-container">
-      <div class="about-text">
-        <h2>About Me</h2>
-        <p>Filler Text</p>
+    <section id="aboutMe" class="tab-content">
+      <div class="about-container">
+        <div class="about-text">
+          <h2>About Me</h2>
+          <p>Born in Colorado, grew up in Louisiana.</p>
+          <p>Been living in Washington since 2001.</p>
+          <p>Went to Western Washington University and graduated with a BS in Environmental Science.</p>
+        </div>
+        <div class="about-pic">
+          <img class="me-pic" src="Images/patrick.jpg">
+        </div>
       </div>
-      <div class="about-pic">
-        <img src="Images/bridge.jpg">
+
+      <div class="skills-container">
+        <h2>Skills</h2>
+        <ul>
+          <li>Javascript</li>
+          <li>Environmental Science</li>
+        </ul>
       </div>
-    </div>
+    </section>
 
-    <section id="projectsContainer">
+    <section id="projectsContainer" class="tab-content">
```

Comment:
**TODO FOR LOST POINTS**

Same thing with taking out camel-casing here as mentioned above.